app/src/main/java/com/muzima/view/setupconfiguration/GuidedConfigurationWizardActivity.java

```diff
@@ -30,10 +30,13 @@
 import com.muzima.R;
 import com.muzima.adapters.ListAdapter;
 import com.muzima.adapters.setupconfiguration.GuidedSetupActionLogAdapter;
+import com.muzima.api.model.Form;
```

Comment:
@samuelowino is this import necessary?

Reply:
This particular one seems to be in use

LoC Ref: 372 - 378